app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os.path
+import shutil
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
```

Comment:
Code Style issue: You are assuming that the command will always be in the correct format. It's better to add a check if the command is a valid `mv` command.